doc/source/users_guide/cime-change-namelist.rst

```diff
@@ -3,24 +3,29 @@
 Customizing your input variables
 ================================
 
-CIME and current CIME-compliant components currently primarily uses fortran namelists.
+CIME and CIME-compliant components currently use Fortran namelists to control runtime options.
```

Comment:
primarily is correct here.   we also have seq_maps.rc and some components do use other than fortran namelists.

Reply:
Ok changing back to primary and adding that some models may use other text files.